dtdl-parser/src/model_parser.rs

```diff
@@ -1101,16 +1095,16 @@ mod model_parser_tests {
 
     #[rustfmt::skip]
     #[test]
-    fn demo_validation_test() {
+    fn sdv_vehicle_validation_test() {
```

Comment:
Probably because you are missing the submodule that provides that DTDL file.

When you clone the repo, make sure that you use the --recurse-submodules option:

`git clone --recurse-submodules https://github.com/eclipse-ibeji/ibeji`